benchmarks/src/copy.cpp

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <benchmark/benchmark.h>
+#include <string.h>
+#include <vector>
+
+#include <udt.hpp>
+
+template <typename Contained>
+static void handwritten_loop(benchmark::State& state) {
+    const size_t r0 = static_cast<size_t>(state.range(0));
+    std::vector<Contained> in_buffer(r0);
+    std::vector<Contained> out_buffer(r0);
+    for (auto _ : state) {

```

Comment:
Do we need to compile in C++14 mode, or could we `[[maybe_unused]]` these instead of casting to `void`?

Reply:
This benchmark is from 2016 so maybe_unused was not a thing yet :D.